libmisc/console.c

```diff
@@ -50,7 +50,7 @@ static bool is_listed (const char *cfgin, const char *tty, bool def);
 static bool is_listed (const char *cfgin, const char *tty, bool def)
 {
 	FILE *fp;
-	char buf[200], *s;
+	char buf[1024], *s;
```

Comment:
Why increase the size?

Reply:
Well, the console path can be theoretically longer than 200 characters. I know I am replacing one arbitrary number with another but 200 characters seems to me to be quite small limit for paths.